Jenkinsfile

```diff
@@ -2,17 +2,19 @@
 
 // PARAMETERS for this pipeline:
 // branchToBuildCTL = refs/tags/20190401211444 or master
-// DESTINATION = user@host_or_ip:/path/to/che-incubator/chectl/
+// DESTINATION = user@host_or_ip:/path/to/che-incubator/chectl
+// BASE_URL = http://host_or_ip/path/to/che-incubator/chectl
 
 def installNPM(){
-	def nodeHome = tool 'nodejs-10.9.0'
+	def nodeHome = tool 'nodejs-10.15.3'
 	env.PATH="${env.PATH}:${nodeHome}/bin"
-	sh "npm install -g yarn pkg"
-	sh "npm version"
+	sh "npm install -g yarn && yarn add -g cpx rimraf oclif && yarn install -f oclif-dev"
```

Comment:
can't you just `npm install -g yarn cpx rimraf oclif oclif-dev`?

Reply:
nope:

```
$➔ npm install -g yarn cpx rimraf oclif oclif-dev
npm ERR! code E404
npm ERR! 404 Not Found: oclif-dev@latest

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/nboldt/.npm/_logs/2019-06-06T13_43_54_517Z-debug.log
```

but:

Reply:
ok so you can regroup `yarn` `cpx` `rimraf` and `oclif` (because they all need to be globally installed) and then `yarn install -f oclif-def`

Reply:
would you prefer 

```
npm install -g yarn && yarn global add cpx rimraf oclif && yarn install -f oclif-dev
```

or

```
npm install -g yarn cpx rimraf oclif && yarn install -f oclif-dev
```

? Seems to me they do the same thing, but :man_shrugging: